CMakeLists.txt

```diff
@@ -414,6 +414,7 @@ if (HCC_INTEGRATE_ROCDL)
             -DLLVM_DIR=${CLANG_BIN_DIR}
             -DAMDHSACOD=${AMDHSACOD}
             -DGENERIC_IS_ZERO=ON
+            -DBUILD_HC_LIB=ON
```

Comment:
@dfukalov there is no such flag `BUILD_HC_LIB` inside `ROCm-Device-Libs`, could you explain the rationale behind adding this line?